packages/common/package.json

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "@wallet-service/common",
+  "packageManager": "yarn@4.1.0",
+  "peerDependencies": {
+    "@aws-sdk/client-lambda": "3.540.0",
+    "@hathor/wallet-lib": "0.39.0",
+    "winston": "^3.13.0"
```

Comment:
question(non-blocking): Here we have a mix of version definitions. Some are pinned to an exact value and some are with a flexible caret range.

Should we decide for one approach or another? Or can we leave this mixed as it currently is?

Reply:
I'll fix all versions in a future PR